### PR TITLE
test: (e2e) add e2e tests for Avatar Group component

### PR DESCRIPTION
**Pull request by AntonOlkhovskyi**
#### Please provide a link to the associated issue.
Closes SAP/fundamental-ngx#5301

#### Please provide a brief summary of this pull request.
add e2e tests for avatar group (core)
#### Please check whether the PR fulfills the following requirements

- [x] the commit message follows the guideline:
https://github.com/SAP/fundamental-ngx/blob/main/CONTRIBUTING.md
- [x] tests for the changes that have been done
- [x] all items on the PR Review Checklist are addressed :
https://github.com/SAP/fundamental-ngx/wiki/PR-Review-Checklist
- [x] Run npm run build-pack-library and test in external application

Documentation checklist:
- [x] update `README.md`
- [x] [Breaking Changes Wiki](https://github.com/SAP/fundamental-ngx/wiki/Breaking-Changes)
- [x] Documentation Examples
- [x] Stackblitz works for all examples

### CODE REVIEW

File: e2e/wdio/core/tests/avatar-group.e2e-spec.ts [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+        click(avatarGroupPage.firstExampleAvatar);
+        waitForPresent(avatarGroupPage.usedDetailsPopup);
+        expect(doesItExist(avatarGroupPage.usedDetailsPopup)).toBe(true);
+
+        scrollIntoView(avatarGroupPage.secondExampleAvatar);
+        click(avatarGroupPage.secondExampleAvatar);
+        waitForPresent(avatarGroupPage.usedGroupDetailsPopup);
+        expect(doesItExist(avatarGroupPage.usedGroupDetailsPopup)).toBe(true);
+    });
+
```
Comment (Seamoo13) [inline e2e/wdio/core/tests/avatar-group.e2e-spec.ts:21]: I think 1-2 more cases are needed
when you click on group type it opens a popup with individual users (you check this now), each user is clickable and opens an individual user card. Should check this. also back btn functionality on this user card

File: e2e/wdio/core/pages/avatar-group.po.ts [hunk 2]
```diff
@@ -0,0 +1,31 @@
+import { CoreBaseComponentPo } from './core-base-component.po';
+import { waitForElDisplayed, waitForPresent } from '../../driver/wdio';
+
+export class AvatarGroupPo extends CoreBaseComponentPo {
+    private url = '/avatar-group';
+    root = '#page-content';
+
+    firstExampleAvatar = '#background-ex0 fd-avatar';
+    // usedDetailsPopup = '.fd-quick-view__content';
```
Comment (Seamoo13) [inline e2e/wdio/core/pages/avatar-group.po.ts:9]: not needed?

**Review by Seamoo13 — CHANGES_REQUESTED** [review]

File: e2e/wdio/core/tests/avatar-group.e2e-spec.ts [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+    const avatarGroupPage = new AvatarGroupPo();
+
+    beforeAll(() => {
+        avatarGroupPage.open();
+    }, 1);
+
+    it('should have details popup on click', () => {
+        click(avatarGroupPage.firstExampleAvatar);
+        waitForPresent(avatarGroupPage.usedDetailsPopup);
+        expect(doesItExist(avatarGroupPage.usedDetailsPopup)).toBe(true);
```
Comment (nikvalor) [inline e2e/wdio/core/tests/avatar-group.e2e-spec.ts]: please message if the element is not exist

File: e2e/wdio/core/tests/avatar-group.e2e-spec.ts [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+
+        scrollIntoView(avatarGroupPage.secondExampleAvatar);
+        click(avatarGroupPage.secondExampleAvatar);
+        waitForPresent(avatarGroupPage.usedGroupDetailsPopup);
+        expect(doesItExist(avatarGroupPage.usedGroupDetailsPopup)).toBe(true);
```
Comment (nikvalor) [inline e2e/wdio/core/tests/avatar-group.e2e-spec.ts]: same as above

**Review by InnaAtanasova — APPROVED** [review]

**Comment by Seamoo13** [comment]
reopened here https://github.com/SAP/fundamental-ngx/pull/5450

**Pull request closed without merging**